packages/scandipwa/src/component/FieldNumber/FieldNumber.container.js

```diff
@@ -74,11 +74,15 @@ export class FieldNumberContainer extends PureComponent {
             attr: { min = 0, max = DEFAULT_MAX_PRODUCTS } = {}
         } = this.props;
 
+        const { value: stateValue } = this.state;
+
         // eslint-disable-next-line no-nested-ternary
         const rangedValue = value < min ? min : value > max ? max : value;
 
-        this.fieldRef.value = value;
-        this.setState({ value: rangedValue });
+        if (Math.sign(stateValue) !== 1) {
```

Comment:
Please make comparisons with comparison operators.
They are shorter and better readable.